data/json/monstergroups/triffid.json

```
@@ -4,30 +4,39 @@
     "name": "GROUP_TRIFFID",
     "default": "mon_triffid",
     "monsters": [
-      { "monster": "mon_triffid_young", "freq": 100, "cost_multiplier": 1 },
+      { "monster": "mon_triffid_young", "freq": 300, "cost_multiplier": 0, "pack_size": [ 2, 4 ] },
       { "monster": "mon_fungal_fighter", "freq": 100, "cost_multiplier": 1 },
+      { "monster": "mon_triffid", "freq": 100, "cost_multiplier": 2, "pack_size": [ 2, 3 ] },
       { "monster": "mon_vinebeast", "freq": 160, "cost_multiplier": 10 },
       { "monster": "mon_triffid_queen", "freq": 60, "cost_multiplier": 20 },
-      { "monster": "mon_creeper_hub", "freq": 50, "cost_multiplier": 2 },
-      { "monster": "mon_biollante", "freq": 50, "cost_multiplier": 2 },
-      { "monster": "mon_triffid_heart", "freq": 0, "cost_multiplier": 0 }
+      { "monster": "mon_biollante", "freq": 200, "cost_multiplier": 2 },
+      { "monster": "mon_triffid_flower", "freq": 100, "cost_multiplier": 4 },
+      { "monster": "mon_biollante", "freq": 50, "cost_multiplier": 2 }
```

```suggestion
      { "monster": "mon_biollante", "freq": 50, "cost_multiplier": 2, "pack_size": [ 2, 3 ] }
```
Otherwise redundant with line above.